### DISCUSSION TITLE
TV SERIES RENAMER NOW AVAILABLE 🚀

### DISCUSSION THREAD

**Discussion by sachinsenal0x64** [sachinsenal0x64/fixarr-old#2] [Ideas]
TV SERIES RENAMER NOW AVAILABLE 🚀

**Comment by SneakyZEKE** [comment]
This application would replace perfectly the only GUI satisfying Docker container I've found to automate the renaming of my videos...and free too!

**Comment by sachinsenal0x64** [comment]
I hope you enjoy it cheers 🥂
